Sources/FluentSQLiteDriver/SQLite+Fluent.swift

```diff
@@ -142,6 +141,14 @@ extension SQLiteRow: SQLRow {
     }
 }
 
+import Foundation
+
+extension SQLiteError: LocalizedError {
+    public var errorDescription: String? {
```

Comment:
Why is this added? `self.description` clearly is not localized, and the users of this library (especially those that want to use it on macOS/iOS) might want to add localization themselves.

Reply:
@broadwaylamb without this, the errors won't show up correctly if thrown at top-level in an XCTestCase. If there's a better way to achieve this, I'm all for it.